java-client-model/src/test/java/com/cumulocity/model/utils/StringReader.java

```diff
@@ -0,0 +1,49 @@
+package com.cumulocity.model.utils;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.Scanner;
+
+import com.google.common.io.CharStreams;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+public class StringReader {
+
+    /**
+     * TODO This implementation should be reviewed. It does not handle encoding explicitly
+     * (UTF-8 should be used for all files), and should either read from file or classpath. 
+     */
```

Comment:
interesting comment... class is very old, I would check for usages

Reply:
TODO